module/VuFind/src/VuFind/Controller/Plugin/FavoritesFactory.php

```diff
@@ -72,7 +72,8 @@ public function __invoke(
             $container->get(\VuFind\Record\Loader::class),
             $container->get(\VuFind\Record\Cache::class),
             $container->get(\VuFind\Tags::class),
-            $container->get(\VuFind\Favorites\FavoritesService::class)
+            $container->get(\VuFind\Favorites\FavoritesService::class),
+            $container->get(\VuFind\Db\Service\PluginManager::class)
```

Comment:
I haven't looked really closely, so maybe this isn't a good idea -- but might it make sense to move more functionality into the FavoritesService helper class, instead of adding a new dependency to the Favorites plugin?